content/blog/metrics-at-imperial.md

```diff
@@ -0,0 +1,121 @@
+---
+author: "Emma Russell"
+date: 2019-03-14
+title: Metrics at Imperial*
+best: false
+---
+
+What metrics should a software team be recording to measure their performance? Should these metrics have a different 
+focus for RSE teams in general, and ourselves in particular? 
+
+In previous jobs in a more commercial setting, I had become accustomed to thinking about 'metrics' in two broad 
+categories - firstly those defining the performance and health of online systems (number of requests served, click 
+through rate, uptime etc), and secondly those measuring the productivity of the team in making additions and 
+improvements to those systems (story point burndown, number of open bugs, feature leadtime). 
+
+The first category could 
+have immediate business consequences - eg affecting a client's decision on whether to renew a contract, or whether a 
```

Comment:
This is really good, clear and well written. 

Very minor note but eg on this line is different to e.g. used below (line 94), probably good to be consistent.

Reply:
Well spotted!